OpenRA.Utility/Program.cs

```diff
@@ -28,7 +28,6 @@ static void Main(string[] args)
 			AppDomain.CurrentDomain.AssemblyResolve += GlobalFileSystem.ResolveAssembly;
 
 			Log.AddChannel("perf", null);
-			Log.AddChannel("debug", null);
```

Comment:
I think because --replay-metadata and --replay-report are the only utility functions that use that? So I moved it to those classes.